data/json/professions.json

```diff
@@ -136,8 +136,43 @@
       { "item": "cookbook_human", "bonus": [ [ "CANNIBAL" ] ] },
       { "item": "teleumbrella", "bonus": [ [ "ALBINO" ] ] }
     ]
-  },
-  {
+  },{
+    "type": "profession",
+    "ident": "vagabond",
+    "name": "Vagabond",
+    "description": "Circumstances left you wandering, with no home, no family, no friends.  But the world you knew is gone, and maybe your experiences relying on yourself to survive could be useful in this new one.",
+    "points": 3,
+    "skills": [
+      { "level": 2, "name": "unarmed" },
+      { "level": 2, "name": "survival" },
```

Comment:
The kind of stuff covered in survival, eg skinning or foraging, just don't fit with the urban street dweller imo